feature/tabsflow/search/src/main/java/com/epmedu/animeal/tabs/search/presentation/ui/SearchScreenUi.kt

```diff
@@ -145,10 +150,31 @@ private fun ScreenScaffold(
             }
         )
     }
-    if (state.feedState.feedingConfirmationState
-        == FeedingConfirmationState.FeedingStarted
-    ) {
-        navigator.navigateTo(TabsRoute.Home.name)
+    OnFeedingConfirmationStateChange(
+        navigator,
+        onFeedingEvent,
+        state.feedState.feedingConfirmationState
+    )
+}
+
+@Composable
+fun OnFeedingConfirmationStateChange(
```

Comment:
This fun can be private, and, I guess, we can remove "change" from the name, so it will become "OnFeedingConfirmationState" matching already existing "OnState"